src/Chunk.ts

```diff
@@ -1019,7 +1021,9 @@ export default class Chunk {
 			if (exportVariable instanceof ExportShimVariable) {
 				this.needsExportsShim = true;
 			}
-			exportVariable.exportName = exportName;
+			if (!exportVariable.exportName || exportVariable.exportName.includes(exportName))
+				exportVariable.exportName = [exportName];
+			else exportVariable.exportName.push(exportName);
```

Comment:
This seems the wrong place for this, as it is happening before safe name assignments as far as I can tell?